protocol/messenger_mailserver.go

```diff
@@ -346,7 +347,7 @@ func (m *Messenger) syncFiltersFrom(filters []*transport.Filter, lastRequest uin
 
 	topicsData := make(map[string]mailservers.MailserverTopic)
 	for _, topic := range topicInfo {
-		topicsData[topic.Topic] = topic
+		topicsData[topic.PubsubTopic+topic.ContentTopic] = topic
```

Comment:
That is interesting. Forgive me for suggesting this but should there be some kind of delimiter between the `topic.PubsubTopic` and the `topic.ContentTopic`?

Reply:
Good idea, let's add a delimiter!